<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWConsultaData</class>
 <widget class="QMainWindow" name="MWConsultaData">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta de Datos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Acciones</string>
    </property>
    <widget class="QPushButton" name="btnCapturar">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>60</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Capturar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEditar">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>101</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>140</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNuevo">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>20</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Nuevo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnActualizar">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>180</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancelar">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>220</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
